openhands/server/listen.py

```diff
@@ -1022,3 +906,122 @@ async def get_response(self, path: str, scope):
 
 
 app.mount('/', SPAStaticFiles(directory='./frontend/build', html=True), name='dist')
+
+app = socketio.ASGIApp(sio, other_asgi_app=app)
+
+
+@sio.event
+async def connect(connection_id: str, environ):
+    logger.info(f'sio:connect: {connection_id}')
+
+
+@sio.event
+async def oh_action(connection_id: str, data: dict):
+    """WebSocket endpoint for receiving events from the client (i.e., the browser).
+    Once connected, the client can send various actions:
+    - Initialize the agent:
+    session management, and event streaming.
+        ```json
+        {"action": "initialize", "args": {"LLM_MODEL": "ollama/llama3", "AGENT": "CodeActAgent", "LANGUAGE": "en", "LLM_API_KEY": "ollama"}}
+
+    Args:
+        ```
+        websocket (WebSocket): The WebSocket connection object.
+    - Start a new development task:
+        ```json
+        {"action": "start", "args": {"task": "write a bash script that prints hello"}}
```

Comment:
I don't think these docs are relevant/accurate anymore, are they? I think maybe we can remove them, they're probably out of date anyways